packages/manager/src/features/LoadBalancers/LoadBalancerCreate/ConfigurationDetails.tsx

```diff
@@ -0,0 +1,81 @@
+import { Grid } from '@mui/material';
+import Stack from '@mui/material/Stack';
+import { useFormikContext } from 'formik';
+import * as React from 'react';
+
+import { Autocomplete } from 'src/components/Autocomplete/Autocomplete';
+import { TextField } from 'src/components/TextField';
+import { TooltipIcon } from 'src/components/TooltipIcon';
+import { Typography } from 'src/components/Typography';
+
+import type { Protocol } from '@linode/api-v4';
+
+const protocolOptions = [
+  { label: 'HTTPS', value: 'https' },
+  { label: 'HTTP', value: 'http' },
+  { label: 'TCP', value: 'tcp' },
+];
+
+interface Props {
+  index: number;
+}
+
+export const ConfigurationDetails = ({ index }: Props) => {
+  const { handleChange, setFieldValue, values } = useFormikContext<{
+    configurations: { label: string; port: number; protocol: Protocol }[];
+  }>();
```

Comment:
Can we go ahead and use the actual type?

```suggestion
  const { handleChange, setFieldValue, values } = useFormikContext<CreateLoadbalancerPayload>();
```

Reply:
Formik is complaining about a missing certificates fields, as it is required in the `configurations`. So, as we build the other parts of the configurations (such as the stepper service-targets and  rounts), we could actually use `CreateLoadbalancerPayload`. But, I'm open to your thoughts on this.